accepted/single-file/design.md

```diff
@@ -101,22 +127,26 @@ On Startup, the [host components](https://github.com/dotnet/core-setup/blob/mast
   
   * Implements a bundle-probe function, which will be used by the runtime to probe contents of the bundle when resolving assemblies. 
   
-    * Bundle probe has the following signature:
+    ```C++
+  /// <summary>
+    /// <param name="path"> Relative-path to the file being probed. </param>
+    /// <param name="size"> Out-param: size of the file, if found. </param>
+    /// <param name="path"> Out-param: offset within the bundle, if found</param>
+    /// <returns> true if the requested file is found in the bundle, 
+    ///           false otherwise. </returns>
+    /// </summary>
+    bool bundle_probe(const char *path, int64_t *size, int64_t *offset);
+    ```
+    
+  * The `bundle_probe` function pointer is passed to the runtime (encoded as a string) through a property named `BUNDLE_PROBE`.
+  
+  * When probing for assemblies, the the [assembly resolution logic](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/docs/design/features/assembly-conflict-resolution.md#probe-ordering) will treat bundled assemblies similar to assemblies in the app directory. The probe ordering will be in the order:  servicing location, shared store, framework directory(s) from higher to lower, the *single-file bundle*, app directory, additional locations specified in runtimeconfig.dev.json file.
```

Comment:
I just updated the doc with the correct probe order (as is in the code): https://github.com/dotnet/runtime/blob/master/docs/design/features/assembly-conflict-resolution.md. Please fix the text in this document to match (app goes before framework, bundle should go before app).

Nit: double `the` on the first line.